src/BooleanFlag.php

```diff
@@ -29,17 +45,81 @@ public function createBooleanFlag(string $key, string $description, array $tags
 
         $body->setKey($key);
         $body->setDescription($description);
-        $body->setTemplate('simple_boolean_flag');
 
         $flag = $this->flagApi->createFlag($body);
+        if ($flag instanceof Error) {
```

Comment:
🤔 It doesn't seem to return a Flagr\Client\Model\Error? The error is set as a response object on ApiException.